torch/lib/c10d/ProcessGroupGloo.cpp

```diff
@@ -757,7 +779,9 @@ class AsyncAllgatherWork : public ProcessGroupGloo::AsyncWork {
   std::vector<at::Tensor> inputs;
   const uint32_t tag;
 
-  void run() override {
+  void allGather(
```

Comment:
allgather in lowercase to match style in c10d.

Reply:
updated.